File: 861-flipping-an-image/flipping-an-image.swift
class Solution {
    func flipAndInvertImage(_ image: [[Int]]) -> [[Int]] {
        var tempArr = [[Int]]()
        for i in image {
            var tempInt = [Int]()
            for j in i {
                tempInt.insert(j, at: 0)
            }
            tempArr.append(tempInt.map { $0 == 1 ? 0 : 1 })
        }
        return tempArr
    }
}

File: 861-flipping-an-image/flipping-an-image.kt
class Solution {
    fun flipAndInvertImage(image: Array<IntArray>): Array<IntArray> {
        for (i in image.indices) {
            image[i] = image[i].reversedArray()
            for (j in image[i].indices) {
                image[i][j] = image[i][j] xor 1
            }
        }

        return image
    }
}